mysite/wsgi.py

```diff
@@ -0,0 +1,10 @@
+"""WSGI config for mysite project."""
+
+
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'mysite.settings')
+# pylint: disable=invalid-name
+application = get_wsgi_application()
```

Comment:
It is better to put comments in the same line if they fit the line length limit.